language/move-stdlib/tests/fixedpoint32_tests.move

```diff
@@ -1,5 +1,5 @@
 #[test_only]
-module std::fixed_point32_tests {
+module std::fp32_tests {
     use std::fixed_point32;
```

Comment:
This change of name is to pass the 64 char limit on external symbol length imposed by the Solana VM.